app/views/backstage/labs/edit.html.haml

```diff
@@ -8,38 +8,33 @@
     .edit-lab
 
     %fieldset
-      %legend Basic Details
-      .row
-        .col-lg-12
-          %h5
-            Name, description and slug of your Lab
-          = f.input :name
-          = f.input :blurb
-          = f.input :description
-          = f.input :slug
-      .row
-        .col-lg-12
-          %h5
-            Kind of Lab
-          %p
-            What kind of Fab Lab is your Lab?
-          - Lab::kinds.each do |value, index|
-            .col-lg-2
-              .radio-inline
-                %span
-                  = f.radio_button :kind, value
-                  %label.radio{style: "width: 160px;"}
-                    = value.humanize.titleize
+      .text-large Basic Details
+      .text-medium
```

Comment:
Why use `text-medium` instead of `h5`?
I think h1-5 is better because we know what it means. It is standard.

Does everyone know what text-medium means?